boilerplate/openshift/golang-osd-operator/TEST_README.md

```
@@ -0,0 +1,226 @@
+
+```bash
+# For bash tests, preserve temp directories
+PRESERVE_TEMP_DIRS=1 ./test/case/convention/openshift/golang-osd-operator/08-pko-migration
+
+# For Python tests, use verbose mode
+python3 -m pytest test_olm_pko_migration.py -vv -s
+```
+
+### Common Issues
+
+**ImportError: No module named 'yaml'**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)